spec/scss_lint/linter/groupable_selector_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+describe SCSSLint::Linter::GroupableSelector do
+
+  context 'when different elements' do
+    let(:scss) { <<-SCSS }
+      p {
+        background: #000;
+        margin: 5px;
+      }
+      a {
+        background: #000;
+        margin: 5px;
+      }
```

Comment:
Are there other examples of situations where this linter is useful? If this is the only case, I'm not sure having this linter is really worth it, as the number of times where you have the exact same set of properties in both selectors is likely quite rare.